{"name": "test_fixture_scope_function02", "status": "failed", "statusDetails": {"message": "AssertionError: assert 85 > 100", "trace": "random_num = 85\n\n    @allure.feature(\"feature: test_02_fixture\")\n    @allure.story(\"story: scope\")\n    def test_fixture_scope_function02(random_num):\n        print(f'random_num: {random_num}')\n>       assert random_num > 100\nE       assert 85 > 100\n\ntest_02_fixture_for_allure.py:128: AssertionError"}, "start": 1627038110016, "stop": 1627038110016, "uuid": "95d94939-b147-46f7-b2ce-c5d418d2f96f", "historyId": "bc793ffa39f480a6e531b46edd1f566b", "testCaseId": "9a54bb79dd6497bb4fcb3429e98858cf", "fullName": "test_02_fixture_for_allure#test_fixture_scope_function02", "labels": [{"name": "story", "value": "story: scope"}, {"name": "feature", "value": "feature: test_02_fixture"}, {"name": "suite", "value": "test_02_fixture_for_allure"}, {"name": "host", "value": "JULIZHOU-6JV5P"}, {"name": "thread", "value": "18748-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_02_fixture_for_allure"}]}